flow/designs/asap7/mock-array-big/config.py

```diff
@@ -1,47 +1,34 @@
 import os
 
-# number of Elements in row and column, user can set via environment variable
-rows = int(os.environ.get("MOCK_ARRAY_HEIGHT"))
-cols = int(os.environ.get("MOCK_ARRAY_WIDTH"))
-
-# Element placement pitch can be control by user
-pitch_scale = int(os.environ.get("MOCK_ARRAY_PITCH_SCALE"))
-
-if pitch_scale < 1:
-  raise Exception("Element placement pitch must be greater than 1")
-
-# Routing pitches for relevant metal layers.
-#  For x, this is M5; for y, this is M4.
-#  Pitches are specified in OpenROAD-flow-scripts/flow/platforms/asap7/lef/asap7_tech_1x_201209.lef.
-#  For asap7, x and y pitch is the same.
-#
-# make_tracks M5 -x_offset 0.012 -x_pitch 0.048 -y_offset 0.012 -y_pitch 0.048
-#
-# the macro needs to be on a multiple of the track pattern
-routing_pitch = 0.048
-
-placement_grid_x = routing_pitch
-placement_grid_y = routing_pitch
-
-# historically pitch_scale 2 was ca 13 um
-ce_width    = int(3 * 2.16 / placement_grid_x) * placement_grid_x * pitch_scale
-ce_height   = int(3 * 2.16 / placement_grid_y) * placement_grid_y * pitch_scale
-ce_margin_x = placement_grid_x * 8
-ce_margin_y = placement_grid_y * 8
+# routing pitch for M4, M5 and M6 tied to placement grid at 2.16
+# therefore, the optimal placement and Element size should be multiple of 2.16
+# set grid x and y
+placement_grid_x     = 2.16
+placement_grid_y     = 2.16
+
```

Comment:
The changes to this file has erased the previous changes in this file that were added to solve various problems, such as adding a routing channel to deal with blockage area + various comments to explain magic numbers.

I think that more of what is in master should be kept (comments and calculations) and if they are deleted, then it should be clearer why it has been deleted.

Reply:
Will do